socket/server.go

```diff
@@ -61,7 +60,12 @@ func (s *Server) Close() error {
 }
 
 func (s *Server) handleConn(conn net.Conn) {
-	defer conn.Close()
+	defer func(conn net.Conn) {
+		err := conn.Close()
+		if err != nil {
+
+		}
+	}(conn)
```

Comment:
Since it's now handler "properly", maybe we can add here `log.Println()` or something?